src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
```

Comment:
I remember seeing somewhere that it was bad practice to use the "__version__" attribute and importing it in `setup.py` since it importing the module executes the code in it. However, I can't remember where it was.

You don't need a __version__ attribute in order to determine the installed version of the package: https://github.com/RPi-Distro/python-gpiozero/issues/381